examples/browser-validation/Makefile

```diff
@@ -0,0 +1,28 @@
+build:
+	docker build -t prio-validate ../../
+
+run:
+	docker run -v `pwd`:/app/examples/browser-validation \
+		-e AWS_ACCESS_KEY_ID \
+		-e AWS_SECRET_ACCESS_KEY \
+		-e PRIO_DATE \
+		-e PRIO_PINGS \
+		-e PRIO_PUBKEY_A \
+		-e PRIO_PVTKEY_A \
+		-e PRIO_PUBKEY_B \
+		-e PRIO_PVTKEY_B \
+		-it prio-validate \
+			sh -c 'cd /app/examples/browser-validation && \
+			pipenv lock && \
+			pipenv sync && \
+			pipenv run python main.py'
+
+test:
```

Comment:
`run` takes a batch of prio-encoded pings and aggregates them. `test` (a test-run) generates keys, data, and encrypted payloads and passes it to the run command.

`run` can be used to validate data from a browser, so that's why it's a separate command.